### ISSUE TITLE
emoji picker creates a bit too much cpu increase

### ISSUE THREAD

**Issue by star-buck** [blue-systems/plasma-5.9#218]
(I'm starting to put all generic, non-valve specific things experienced and possibly also reported elsewhere here. If a proper bugreport already exist, just post a link to it in the comment).

see also:
https://www.reddit.com/r/kde/comments/f44ybw/emoji_picker_is_very_slow_and_causes_high_cpu/

**Comment by star-buck** [comment]
Also this for context:
https://www.reddit.com/r/kde/comments/f27ibv/comment/fhc5ie0

**Comment by davidedmundson** [comment]
Upstream 417454

There are patches up already

**Comment by star-buck** [comment]
Great.

**Comment by Pointedstick** [comment]
See https://phabricator.kde.org/D27337 and https://phabricator.kde.org/D27346

**Comment by aleixpol** [comment]
Should be fixed as of this commit (Plasma/5.18)

```
commit 91c446e0c4430289369963bf44e64228add67d18
Author: Aleix Pol <aleixpol@kde.org>
Date:   Wed Feb 19 19:01:20 2020 +0100

    emojier: Only fit text when we know it's a very complex emoji
    
    BUG: 417454
```

**Comment by Pointedstick** [comment]
Yep, fixed now. Down to 0-1% on average.
